pkg/virt-controller/watch/ovm.go

```diff
@@ -364,7 +364,7 @@ func (c *OVMController) getMatchingControllers(vm *virtv1.VirtualMachine) (ovms
 func (c *OVMController) addVirtualMachine(obj interface{}) {
 	vm := obj.(*virtv1.VirtualMachine)
 
-	log.Log.Object(vm).Infof("VM ADDED: %s", vm.Name)
+	log.Log.Object(vm).Infof("VM ADDED: %s/%s", vm.Namespace, vm.Name)
```

Comment:
The "Object()" call makes sure that we log the identity of the VM. "VM added" should be good enought. Also you should either remove that log message, or drastically incrase the verbosity level to not swamp the logs.